app/views/directives/workflow-arrow-buttons.js

```diff
@@ -630,8 +630,10 @@ const toasterMessages = mergeTranslationKeys(toasterMessagesTranslations);
                         $scope.updateStep(tab);
                     
                     if(tab == "intro"){
-                        loadArticle();
-                        $scope.disablePreviousBtn = true;
+                        $timeout(function() {
+                            loadArticle();
+                            $scope.disablePreviousBtn = true;
+                        },0);
```

Comment:
can alos use $scope.$applyAsync(). will do same thing as `$timeout(, 0)` but using timeout is code smell and implies bad programming 
https://en.wikipedia.org/wiki/Code_smell

```suggestion
                        $scope.$applyAsync(function() {
                            loadArticle();
                            $scope.disablePreviousBtn = true;
                        });
```